.github/scripts/update_overlay_hashes.sh

```
@@ -0,0 +1,115 @@
+set -euo pipefail
+
+if [[ "$#" -lt 2 ]]; then
+  echo "Usage: $0 <overlay_file> <key[[@occurrence]]=flake_output> [<key[[@occurrence]]=flake_output> ...]" >&2
+  exit 1
+fi
+
+readonly OVERLAY_FILE="$1"
+shift
+readonly MAPPINGS=("$@")
+readonly FAKE_HASH='sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA='
+readonly LOG_DIR="$(mktemp -d)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)